### ISSUE TITLE
Default parameter displayed in the analytics view

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/WebClient-Framework#328]
Issue detected on v2.3.0.

Even if the plot of the default parameter (for a given product type) is removed from the analytics view, if the user adds a new product type (i.e. from the layers list) the old default parameter is added again:

1. Access the web interface and click Reset -> MAG LR, Swarm A, F vs Latitude is displayed in the analytics view
2. Change the parameter on the Y axis from F to B_C -> B_C vs Latitude is displayed.
3. Add Plasma data -> the Ne vs Latitude and F vs Latitude plots is added to the analytics view even if F vs Latitude was previously removed.

The same issue occurs also with other product types, e.g.: Plasma data (Te vs Latitude) adding IBI data (Ne vs Latitude is displayed even if it was previously removed)

**Comment by santilland** [comment]
Thanks for the input,
just to make sure the issue is understood correctly, the expected behavior is when adding another collection keep the previously configured parameters.
So in your example it would mean show Ne(plasma) in one plot and B_C (mag) on the other, correct?
What should be the behavior if the x axis is changed?

**Comment by lmar76** [comment]
Correct. If the x axis is changed, I think the GUI should display the default parameter of the new product (e.g. Ne if you add plasma) vs the new parameter on x axis.
